home.admin/config.scripts/bitcoin.update.sh

```diff
@@ -137,7 +137,7 @@ if [ "${mode}" = "tested" ]||[ "${mode}" = "reckless" ]||[ "${mode}" = "custom"
   
   displayInfo
 
-  if [ $installedVersion = $bitcoinVersion ];then
+  if [ $installedVersion = $bitcoinVersion ]; then
```

Comment:
adding a space won't make any difference there.
Can add quotes to avoid the error when the variable is not defined:
```suggestion
  if [ "$installedVersion" = "$bitcoinVersion" ]; then
```